src/CW/Util/Request.php

```diff
@@ -47,6 +47,26 @@ public static function current() {
   }
 
   /**
+   * Checks if the current request is a POST.
+   *
+   * @return bool
+   */
+  public function isPOST() {
+    return $_SERVER['REQUEST_METHOD'] === 'POST';
+  }
+
+  /**
+   * Uses all the options to check if the current request is HTTPS.
+   *
+   * @return bool
+   */
+  public function isHTTPS() {
+    $is_https = isset($_SERVER['HTTPS']) && $_SERVER['HTTPS'] == 'on';
+    $is_https_forwarded = isset($_SERVER['HTTP_X_FORWARDED_PROTO']) && $_SERVER['HTTP_X_FORWARDED_PROTO'] == 'https';
```

Comment:
But this ($_SERVER['HTTP_X_FORWARDED_PROTO'] == 'https') doesn't match if it's zero, does it?